### PR TITLE
Add replacements for _.first and _.last

### PR DESCRIPTION
**Pull request by minijus**

### CODE REVIEW

**Comment by minijus** [comment]
Hi, @stevemao,

could you look into this PR and merge it if all is ok?

File: lib/rules/rules.json [hunk 1]
```diff
@@ -34,6 +38,10 @@
         "alternative": "Array.prototype.find()",
         "ES6": true
     },
+    "first": {
+        "compatible": true,
+        "alternative": "Array[0] or Array.prototype.slice()"
```
Comment (stevemao) [inline lib/rules/rules.json]: This should just be `Array.prototype.slice()`

**Comment by stevemao** [comment]
@cht8687 WDYT?

File: lib/rules/rules.json (hunk 1 above)
Comment (minijus) [reply lib/rules/rules.json]: Updated

**Comment by cht8687** [comment]
@stevemao  LGTM

**Pull request merged**